chromadb/test/conftest.py

```diff
@@ -183,26 +279,62 @@ def sqlite_persistent() -> Generator[System, None, None]:
 
 
 def system_fixtures() -> List[Callable[[], Generator[System, None, None]]]:
-    fixtures = [fastapi, fastapi_persistent, sqlite, sqlite_persistent]
+    fixtures = [
+        fastapi,
+        fastapi_persistent,
+        sqlite,
+        sqlite_persistent,
+        fastapi_server_auth,
+    ]
     if "CHROMA_INTEGRATION_TEST" in os.environ:
         fixtures.append(integration)
     if "CHROMA_INTEGRATION_TEST_ONLY" in os.environ:
         fixtures = [integration]
     return fixtures
 
 
+def system_fixtures_auth() -> List[Callable[[], Generator[System, None, None]]]:
+    fixtures = [
+        fastapi_server_auth_param,
+        fastapi_server_auth_file,
+        fastapi_server_auth_env,
+    ]
+    return fixtures
+
+
 @pytest.fixture(scope="module", params=system_fixtures())
 def system(request: pytest.FixtureRequest) -> Generator[API, None, None]:
     yield next(request.param())
 
 
+@pytest.fixture(scope="module", params=system_fixtures_auth())
+def system_auth(request: pytest.FixtureRequest) -> Generator[API, None, None]:
+    yield next(request.param())
+
+
 @pytest.fixture(scope="function")
 def api(system: System) -> Generator[API, None, None]:
     system.reset_state()
     api = system.instance(API)
     yield api
 
 
+@pytest.fixture(scope="function")
+def api_wrong_cred(
```

Comment:
as a rule, we only add fixtures to this file if they are intended to be shared across tests.

I think we can encapsulate all the testing for this into one test file and maybe just want to share the logic there?

Reply:
I will create a new subdir for all auth tests as those are expected to grow with additional auth providers